pyproject.toml

```diff
@@ -13,7 +13,7 @@ description = "Fortran to Python interface generator with derived type support"
 authors = [{name = "James Kermode", email = "james.kermode@gmail.com"}]
 python-requires = ">=3.8"
 urls = {Homepage = "https://github.com/jameskermode/f90wrap"}
-dependencies = ["numpy>=1.13", "packaging"]
+dependencies = ["numpy>=1.13", "packaging", "pytest", "pytest-dependency"]
```

Comment:
Two questions here:

1) Should `pytest-dependency` really be used? Avoiding dependencies between tests is usually considered good practice.
2) If 1) can be avoided, is `pytest` even necessary, or better stick to good old `unittest` (I rewrote mine after realizing that `pytest` is not a requirement)?